views/calendar.erb

```diff
@@ -95,7 +99,7 @@
                      </tr>
                   <% end %>
                   <%rowNum += 1%>
-               <%end%>
+               <%end%>            
```

Comment:
Style nitpick: avoid trailing whitespace

Reply:
like above, will fix at build today or earlier

Reply:
Also fixed in 1ac702b, should be done now.